modules/node/persistence/cms_node_storage_sql_page_extension.e

```diff
@@ -138,7 +138,7 @@ feature {NONE} -- Implementation
 
 feature -- SQL
 
-	sql_select_node_data: STRING = "SELECT nid, revision, parent FROM page_nodes WHERE nid =:nid AND revision<=:revision ORDER BY revision DESC LIMIT 1;"
+	sql_select_node_data: STRING = "SELECT nid, revision, parent FROM page_nodes WHERE nid =:nid AND revision =:revision ORDER BY revision DESC LIMIT 1;"
```

Comment:
This is not the good correction to the issue we discussed before.
since this change would break other node_data query.
For reminder, the fix was to change the computation of l_update to be
    l_update := a_node.revision = d.revision